lib/events-native.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @fileoverview
+ * @author Box
+ */
+
+Box.Events = (function(){
+    'use strict';
+
+    return {
+        on: function(element, type, listener) {
+            element.addEventListener(type, listener);
```

Comment:
useCapture was only made optional for modern browsers I think. Should do addEventListener(type, listener, false) for compatibility. Or maybe pass it in? If its optional in IE9+ then maybe we can ignore it.